lib/ui/screens/products/components/paged_hits_grid_view.dart

```diff
@@ -5,11 +5,15 @@ import 'package:infinite_scroll_pagination/infinite_scroll_pagination.dart';
 
 class PagedHitsGridView extends StatelessWidget {
   const PagedHitsGridView(
-      {Key? key, required this.pagingController, this.onHitClick})
+      {Key? key,
+      required this.pagingController,
+      this.onHitClick,
+      this.onItemsFound})
       : super(key: key);
 
   final PagingController<int, Product> pagingController;
   final Function(String)? onHitClick;
+  final WidgetBuilder? onItemsFound;
```

Comment:
I suppose there is a typo here :) 
```suggestion
  final WidgetBuilder? noItemsFound;
```

Reply:
Yes, nice catch!